scripts/appConfig.ts

```diff
@@ -20,5 +20,21 @@ if (appConfig.ui == null) {
 }
 
 export const dashboardRoute = '/workspace';
+export const IDENTITY_KEY = 'sess:user';
+
+export function getUserInterfaceLanguage() {
+    const user: IUser | null = JSON.parse(localStorage.getItem(IDENTITY_KEY));
+    const language = user?.language ?? appConfig.language ?? window.navigator.language ?? 'en';
+
+    if (appConfig.profileLanguages.includes(language)) {
+        return language;
+    } else {
+        return 'en';
```

Comment:
there is also [default_language](https://github.com/superdesk/superdesk-client-core/blob/develop/scripts/core/superdesk-api.d.ts#L1070) we might use here

Reply:
> language: string; // default client language

we should probably remove one. Do you know what's the difference between `language` and `default_language`?

Reply:
`default_language` is set on server, so would make sense to use this one

Reply:
I'll drop the `language` from the interface then since I don't see it used anywhere else but in old `gettext` setup I removed. I didn't find any references in `superdesk.config.js` in customer repos either.